source/best-practices/backup.rst

```diff
@@ -1,61 +1,158 @@
 备份与还原
 ==========
 
+:本节贡献者: |田冬冬|\（作者）、
+             |姚家园|\（作者）
+:最近更新日期: 2021-01-30
+:预计阅读时间: 20 分钟
+
+.. warning::
+
+   本章正在编写中，尚未完成。
+
+----
+
 **备份！备份！备份！**
 
 尽管不经常发生，但电脑随时可能会坏掉或被盗，电脑硬盘也随时可能会出问题，
-也可能不小心使用一个命令把几个月的工作误删了。所以，备份非常重要，不进行必要的备份
-是十分危险的。
+也可能不小心使用一个命令（如 ``rm *``）把几个月的工作误删了。所以，备份非常重要，不进行必要的备份
+是十分危险的。笔者和一些同行多次体验过数据误删或硬盘损坏又未备份带来的痛苦。
+
+.. warning::
 
-备份的目的
+   日常科研工作中，电脑硬盘经常进行大量的读写操作，直接影响硬盘的寿命。一般而言，电脑硬盘的平均
+   寿命大约 5 年左右。当然，硬盘的品质和使用情况不同，其寿命也有所不同。
+
+备份的策略
 ----------
 
+备份的策略简单概括有三种：
+
 - 仅备份重要文件
 - 仅备份家目录
 - 全盘备份
 
+不同备份策略的主要区别是所需的硬盘空间和备份时间，以及硬盘摔坏带来的损失。
+
+.. table:: 不同备份策略的比较
+   :align: center
+
+   ================ ====================================================== ===============
+   备份的策略         优点                                                   缺点
+   ================ ====================================================== ===============
+   仅备份重要文件     所需硬盘空间较小、所需备份时间较短                          仅重要文件备份了，若硬盘摔坏，其他文件需要重新获取
+   仅备份家目录       科研和日常文件基本在家目录下，硬盘摔坏带来的损失很小          所需硬盘空间较大、所需备份时间较长
+   全盘备份          硬盘摔坏带来的损失几乎为零                                 所需硬盘空间最大、所需备份时间最长
+   ================ ====================================================== ===============
+
+建议用户根据个人需求和存储空间来决定具体的备份策略。如果\ **文件非常重要**\ ，
+丢失后\ **恢复难度极大**\ 或\ **恢复时间极长**\ ，强烈推荐备份一下，如毕业论文、脚本、代码、
+无法再次获取的数据、经过长时间处理得到的资料。如果拥有更多硬盘存储空间且备份时间能够接受，
+可以进一步选择备份家目录，甚至全盘备份。
```

Comment:
Considering that external hard drivers are so cheap now. Perhaps we should always recommend users to backup the home directory?

Reply:
Yes to me.

Reply:
Then, we may even not need to mention netdisk backup.

Reply:
Yes, netdisk is mainly used to sync files among different devices, right?

Reply:
Yes to me.

Reply:
removed.

Reply:
See update in 3c27bc9.